Expensify/App
# Update Paypal icon to remove white bg

### Details
Paypal icon has the white background, now updated with new icon given [here](https://github.com/Expensify/App/issues/5581#issuecomment-930255443)

### Fixed Issues

$ https://github.com/Expensify/App/issues/5581

### Tests & QA Steps
1. Open the App
2. Go to Settings -> Payments
3. Hover over/Select the Paypal method or Paypal method option in add payment options. 
4. Should not see the white background behind the icon

### Tested On


### Screenshots


#### Web
<img width="1440" alt="Web-01" src="https://user-images.githubusercontent.com/85645967/136105057-44c7733c-d470-458f-b894-13a7bd6b357f.png">

#### Desktop
<img width="1191" alt="Desktop-02" src="https://user-images.githubusercontent.com/85645967/136104749-129f6004-7007-45c6-9a47-2b3cf2b9c2fd.png">

#### Mobile Web
https://user-images.githubusercontent.com/85645967/136104735-8f84de56-abc5-42df-acb5-517839d2f649.mp4

#### iOS
https://user-images.githubusercontent.com/85645967/136104727-be7fe1dd-ca8d-4686-8c23-5443e6158eac.mp4

#### Android
https://user-images.githubusercontent.com/85645967/136104745-6c13e13d-62bc-470a-919c-384ba65c06eb.mov
